right-for-life_front/src/rootReducer.js

```diff
@@ -1,6 +1,7 @@
 import { combineReducers } from 'redux';
 import { fetchDataReducer } from './hoc/withFetchDataIndicators';
+import { useOrNotGoBackReducer as useOrNotGoBack } from './components/SiteContentContainer/useOrNotGoBackReducer';
```

Comment:
[optional] You can create a separate "index.js" file in "SiteContentContainer" folder and export all files through it, which will shorten the path to the function.

Reply:
Done